docs/user-guide.rst

```diff
@@ -253,11 +253,11 @@ certificate verification on Python 2 will be installed::
     pip install urllib3[secure]
 
 If you want to install the packages manually, you will need ``pyOpenSSL``,
-``ndg-httpsclient``, ``pyasn``, and ``certifi``.
+``cryptography``, ``idna``, and ``certifi``.
 
-.. note:: If you are not using Mac OS X or Windows, note that pyOpenSSL depends
-    on `cryptography <https://cryptography.io/en/latest/>`_ which requires
-    additional system packages to compile. See `building cryptography on Linux
+.. note:: If you are not using Mac OS X or Windows, note that `cryptography
```

Comment:
Minor nit, but we should either use the present brand name (OS X), or the new brand name for this year (macOS). Probably the latter. Mac OS X was deprecated in 2012 with the Mountain Lion release.